sklearn/utils/sparsefuncs_fast.pyx

```diff
@@ -156,13 +176,20 @@ def csc_mean_variance_axis0(X):
     variances : float array with shape (n_features,)
         Feature-wise variances
 
+    sum_weights : float array with shape (n_features,)
```

Comment:
If we can update the returns type of this signature and the other one in the diff

```suggestion
    sum_weights : ndarray shape (n_features,), dtype=floating
```